topic-operator/src/main/java/io/strimzi/operator/topic/BaseKafkaImpl.java

```diff
@@ -152,10 +152,10 @@ public MetadataWork(KafkaFuture<TopicDescription> descFuture,
                 result = future.get();
                 LOGGER.trace("Future {} has result {}", future, result);
             } catch (ExecutionException e) {
-                LOGGER.debug("Future {} threw {}", future, e.toString());
                 if (e.getCause() instanceof UnknownTopicOrPartitionException) {
                     result = null;
                 } else {
+                    LOGGER.debug("Future {} threw {}", future, e.toString());
```

Comment:
why the exception is logged only in this `else`. Anyway, it was raised, or?

Reply:
Because we're expecting the `UnknownTopicOrPartitionException`, and we return null in that case, but all other exceptions we treat as an error.